.github/workflows/release-checker.yml

```diff
@@ -0,0 +1,96 @@
+name: "Check for releases"
+on:
+  schedule:
+    - cron: '*/30 * * * *'
```

Comment:
not sure what you mean by merged "back". If you mean just merged, then it will update the PR if there are any changes.

Reply:
Is there a way we could check if the PR-to-update-the-latest-version exists before downloading and installing binaries? Otherwise, it seems like it would download both the go-algorand and indexer binaries every 30 minutes to run the script until the docs get merged. Docs are merged manually, so I don't expect it to be done asap after they are generated.

Reply:
if we don't check this automatically we might miss any changes. Usually devrel is very quick to merge PRs, but even if they take a day or few i don't think it's a big deal, this doesn't cost us anything.